docs/sources/datasources/elasticsearch.md

```diff
@@ -11,6 +11,17 @@ weight = 325
 Grafana ships with advanced support for Elasticsearch. You can do many types of simple or complex Elasticsearch queries to
 visualize logs or metrics stored in Elasticsearch. You can also annotate your graphs with log events stored in Elasticsearch.
 
+Supported Elasticsearch versions:
```

Comment:
We should list only the supported versions here. Also, I am not sure if we explain what "experimental" is.

Reply:
@Elfo404 to what extend is the 8.0+ experimental? Could we remove `experimental` word or do you have anny clarification we could add in mind?

Reply:
> We should list only the supported versions here.

We are listing supported versions. Even though some are deprecated, you can still fully use them.

Reply:
I thought that `Deprecated versions (v2.0+, v5.0+, v6.0+, and v7.0-v7.9+) will be removed in the next major release.` kind of explains it. Deprecated versions are generally still usable. So I think it should be clear.

Reply:
it was marked as experimental because it when we first added support for 8.0 it was still in beta. after that i didn't really check breaking changes (or other things that needs to be done in the plugin itself)